lib/index.js

```diff
@@ -16,9 +16,12 @@ var Metrics = State.extend({
   },
   initialize: function() {
     // initalize the trackers (but disabled by default)
-    _.each(trackers, function(Tracker) {
-      this.trackers.add(new Tracker());
-    }.bind(this));
+    _.each(
+      trackers,
+      function(Tracker) {
+        this.trackers.add(new Tracker());
+      }.bind(this)
+    );
```

Comment:
Optional: To make future reviews easier, don't include what looks like whitespace changes unless they clearly trigger a linter error or similar.

Reply:
will try to avoid next time.